app/views/dashboard/arms/create.js.coffee

```diff
@@ -28,6 +28,10 @@ $("#sub_service_request_header").html("<%= escape_javascript(render(partial: 'da
 $("#subsidy_information").html("<%= escape_javascript(render(partial: 'dashboard/subsidies/subsidy', locals: { sub_service_request: @sub_service_request, admin: true })) %>");
 $(".selectpicker").selectpicker()
 
+# refreshes calendar
```

Comment:
You are repeating yourself a lot here. Any chance this could be refactored into a JS function?